superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/dndControls.tsx

```diff
@@ -80,7 +80,7 @@ export const dndEntity: typeof dndGroupByControl = {
 export const dnd_adhoc_filters: SharedControlConfig<'DndFilterSelect'> = {
   type: 'DndFilterSelect',
   label: t('Filters'),
-  default: null,
+  default: [],
```

Comment:
Do we also need to update the non-DnD control? https://github.com/apache/superset/blob/299b5dc64448d04abe6b35ee85fbd2b938c781bc/superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/index.tsx#L461

Reply:
Done, tested and works fine (old charts and new charts)